pywr/recorders/_recorders.pyx

```diff
@@ -492,13 +496,17 @@ cdef class NumpyArrayNodeRecorder(NodeRecorder):
         cdef int i
         cdef Timestep ts = self.model.timestepper.current
         for i in range(self._data.shape[1]):
-            self._data[ts.index, i] = self._node._flow[i]
+            self._data[ts.index, i] = self._node._flow[i]*self.factor
         return 0
 
     property data:
         def __get__(self, ):
             return np.array(self._data)
 
+    property factor:
```

Comment:
As discussed this property is unnecessary, and the reason the build is failing. It has the same name (`factor`) as the extension type attribute (in the .pxd). Making factor a `cdef public float` is sufficient to allow access from Python space in this case.